### PR TITLE
Doc fixes

### PR DESCRIPTION
**Pull request by glicht**
## Status
Ready

## Related Issues
fixes: link to the issue

## Description
Fix for relative links with `<img..` tag.

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

**Comment by glicht** [comment]
@ShahafBenYakir small fix to faq doc

**Review by ShahafBenYakir — APPROVED** [review]
Only reviewed the changes in the FAQ article, I assume the rest are part of the infra related to the site build.

**Comment by glicht** [comment]
@Itay4 fix for broken relative image link can be seen here: https://pull-request-473--demisto-content-docs.netlify.app/docs/reference/integrations/proofpoint-protection-server-v2

File: README.md [hunk 1]
```diff
@@ -41,10 +41,23 @@ This task will checkout the content repository and generate the docs. The genera
...
+```bash
  CONTENT_REPO_DIR=~/dev/demisto/content npm run reference-docs
 ```
-
+Additional env vars that affect the generation of the docs:
+* `FILE_REGEX`: specify a regex to select which docs to generate docs for. For example: 
+  ```bash
+  FILE_REGEX=Proofpoint CONTENT_REPO_DIR=~/dev/demisto/content npm run reference-docs
+  ```
+* `MAX_FILES`: max files to generate in each category. Set this for faster generation when developing. Note also in CI builds on branches this is automatically set to speed up the preview site if there is only modifications related to docs. For example:
```
Comment (Itay4) [inline README.md:52]:
```suggestion
* `MAX_FILES`: max files to generate in each category. Set this for faster generation when developing. Note also in CI builds on branches this is automatically set to speed up the preview site if there are only modifications related to docs. For example:
```

**Review by Itay4 — APPROVED** [review]

**Pull request merged**